webapp/app/translations/de/LC_MESSAGES/messages.po

```diff
@@ -5393,3 +5394,8 @@ msgstr "Religion"
 msgid "form.unlock_code.back-to-lotse"
 msgstr "Zurück zur Steuererklärung"
 
+#~ msgid "form.lotse.filing.header-title"
+#~ msgstr ""
+#~ "Steuerformular - Abgabebestätigung - Der "
+#~ "Steuerlotse für Rente und Pension"
+
```

Comment:
Can we delete form.lotse.filing.header-title?

Reply:
Yes we can! :)